configure.ac

```
@@ -93,6 +93,24 @@ if test "x$squid_host_os" = "solaris" -a "x$GCC" != "x" ; then
 	AC_USE_SYSTEM_EXTENSIONS
 fi
 
+AC_MSG_CHECKING([simplified compiler])
```

When I see `checking for simplified compiler...`, I am thinking that ./configure is looking for a compiler that is simpler than some default or standard one. I suggest to avoid this confusion by changing this text. For example:

```suggestion
AC_MSG_CHECKING([compiler name-version])
```

---

Good for me; we have 

AC_MSG_CHECKING([simplified host os])

just a few lines above, I mimicked it. We may want to change it in the future as well